### DISCUSSION TITLE
cancel repeating notification in .NET MAUI

### DISCUSSION THREAD

**Discussion by 99067718** [thudugala/Plugin.LocalNotification#307] [Q&A] [answered]
I made my application repeat a notification once in a while (for certain reasons)
But now I want to know how to stop the loop, since this will get annoying for costumers if you can't stop it.
(at least I think its repeating since I don't want to test it without knowing how to stop it)

(in .NET MAUI)

**Comment by thudugala** [comment, marked as answer]
@99067718 you can cancel the notification LocalNotificationCenter.Current.Cancel(new int[] { notified });
